pkg/core/mempool/hashmap.go

```diff
@@ -141,6 +141,27 @@ func (m *HashMap) Get(txID []byte) transactions.ContractCall {
 	return txd.tx
 }
 
+// Delete a key in the hashmap.
+func (m *HashMap) Delete(txID []byte) {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
+	var k txHash
+
+	copy(k[:], txID)
+
+	m.txsSize -= uint32(m.data[k].size)
```

Comment:
Would you also check if `m.data` contains `k`?

Reply:
Sorry. Forgot to add this to the snippet.

Reply:
Of course, good catch, missed that myself :)

Reply:
Updated!